CONTRIBUTOR-LADDER.md

```diff
@@ -129,16 +130,50 @@ The process for an Organization Member to become a Reviewer is as follows:
 
 Automated tooling assigns PRs across the Reviewers defined for each particular area. 
 
+### Core Scope Repository Committer
+
+Core Scope Repository Committers have write, merge, and voting privileges, and are collectively responsible for steering the repo they are contributing to in a positive direction. As such it represents a significant level of trust in an individual's commitment to working with other committers and the community at large for the benefit of the applicable project.
+
+Core Scope Repository Committers are similar to Committers but only within the scope of the Core Scope Repository they contribute to. Note that Core Scope Repository Committers are not granted the status of Cilium "maintainer" in CNCF parlance. Learn mode about the Core Scope in [REPOSITORY-GUIDELINES.md](REPOSITORY-GUIDELINES.md#core) and see a list of Core Scope repositories in [REPOSITORIES.md](REPOSITORIES.md#core-scope).
+
+Core Scope Repository Committers have all the rights and responsibilities of a Reviewer, plus:
+
+* **Responsibilities:**
+
+    * Mentoring new Core Scope Repository Committers
+    * Discussing strategy and policy for the project they are contributing to
+    * [Voting](https://github.com/cilium/community/blob/main/GOVERNANCE.md#voting) on project matters when required
+    * Additional responsibilities specific to any [project roles](CONTRIBUTOR-ROLES.md) they may have
+    * Approve contributors to assume roles
+
+* **Qualifications:**
+
+    * Laid out in [granting commit access](https://github.com/cilium/community/blob/main/GOVERNANCE.md#granting-commit-access)
```

Comment:
I think we might need to write up something different here because right now the process is too tightly tied to become a Committer

Reply:
What do you think about saying that unless the sub-project has more specific instructions, this is the the default instructions?

Then we can separately pursue what we think is a reasonable path for each subproject, and how we document that clearly for future readers of this ladder.